cdap-api/src/main/java/co/cask/cdap/api/app/AbstractApplication.java

```diff
@@ -170,4 +173,16 @@ protected void scheduleWorkflow(Schedule schedule, String workflowName) {
   protected void scheduleWorkflow(Schedule schedule, String workflowName, Map<String, String> properties) {
     configurer.addSchedule(schedule, SchedulableProgramType.WORKFLOW, workflowName, properties);
   }
+
+
+  /**
+   * Schedule the specified {@link Workflow}
```

Comment:
Only workflow can be scheduled. That enum is for what the program types are allowed to be in a workflow (its a bit confusingly-named).